### DISCUSSION TITLE
记录一次优化 Xcode 编译的过程 | Alex Lin's Notes

### DISCUSSION THREAD

**Discussion by giscus** [chaoskyme/chaoskyme.github.io#325] [Comment]
# 记录一次优化 Xcode 编译的过程 | Alex Lin's Notes

接触新项目后，发现没有改代码的情况下，每次编译基本上编译时间都在一分钟左右。就有了一个想法去解决这个问题，断断续续花了三天时间解决，解决过程中，学习到很多，记录下来。 0x01 发现问题开启编译耗时显示打开终端执行以下命令并重启Xcode： 1$ defaults write com.apple.dt.Xcode ShowBuildOperationDuration -bool YES  编译 B

https://chaosky.tech/2020/04/20/optimize-xcode-build-time/

**Comment by Lianghuajian** [comment]
感谢分享 install! 'cocoapods', :disable_input_output_paths => true
其实还可以解决目录太长等导致的 Argument list too long 编译报错的问题
